src/getrepos.ts

```diff
@@ -8,29 +9,36 @@ export async function getRepoIds(repoPath: string, clientInput: {account, access
   const j1Client = await getClient(clientInput);
 
   for (const repo of repos) {
+    if (repo === '.DS_Store') {
```

Comment:
Note: it may be better to perform some kind of general directory check (maybe `fs` has something?), just to make sure it ignores anything that's not a directory without having to explicitly add a check here.